src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.GraphQL.SystemTextJson.approved.txt

```diff
@@ -0,0 +1,32 @@
+namespace GraphQL.SystemTextJson
+{
+    public class DocumentWriter : GraphQL.IDocumentWriter
+    {
+        public DocumentWriter() { }
+        public DocumentWriter(bool indent) { }
+        public DocumentWriter(System.Text.Json.JsonSerializerOptions options) { }
+        public System.Threading.Tasks.Task WriteAsync<T>(System.IO.Stream stream, T value) { }
+    }
+    public class ExecutionResultJsonConverter : System.Text.Json.Serialization.JsonConverter<GraphQL.ExecutionResult>
+    {
+        public ExecutionResultJsonConverter() { }
+        public override GraphQL.ExecutionResult Read(ref System.Text.Json.Utf8JsonReader reader, System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options) { }
+        public override void Write(System.Text.Json.Utf8JsonWriter writer, GraphQL.ExecutionResult value, System.Text.Json.JsonSerializerOptions options) { }
+    }
+    public class ObjectDictionaryConverter : System.Text.Json.Serialization.JsonConverter<System.Collections.Generic.Dictionary<string, object>>
+    {
+        public ObjectDictionaryConverter() { }
+        public override System.Collections.Generic.Dictionary<string, object> Read(ref System.Text.Json.Utf8JsonReader reader, System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options) { }
+        public override void Write(System.Text.Json.Utf8JsonWriter writer, System.Collections.Generic.Dictionary<string, object> value, System.Text.Json.JsonSerializerOptions options) { }
+    }
+    public static class SchemaExtensions
+    {
+        public static System.Threading.Tasks.Task<string> ExecuteAsync(this GraphQL.Types.ISchema schema, System.Action<GraphQL.ExecutionOptions> configure) { }
+    }
+    public static class StringExtensions
+    {
+        public static System.Collections.Generic.Dictionary<string, object> ToDictionary(this string json) { }
+        public static GraphQL.ExecutionResult ToExecutionResult(this string json, GraphQL.ExecutionErrors errors = null) { }
```

Comment:
@sungam3r did you decide what I should do with this? Dupe into the two test projs that use it or leave it here?

Reply:
I prefer to move it to shared test code: https://github.com/graphql-dotnet/graphql-dotnet/pull/1512#discussion_r374824711

Reply:
Ah sorry, missed that response. Done.